reports/20191212-labitconf-montevideo-uruguay.md

```diff
@@ -0,0 +1,102 @@
+
+## No Booth
+
+Not having a booth turned up nicely. At the event, we were the ones actively talking to people, no matter if they were walking around or at their own booth. This also gave us freedom to stay and talk where people were. Be that the first floor, the second one, or next to the main stage. We strongly suggest ALWAYS considering this kind of approach, unless there are planned activities for the booth like lightning talks, Decred 101 or QA. The booth should not be there waiting for someone to come, most of the team should be walking around engaging with others. Especially at costly events where the booth cost could be compared to hundreds of hours of development.
+
+Overall, not having a booth allowed for the team to be free to go around the conference networking with attendees without having to be attached to a physical booth. On the one hand this strategy was perfect as we were able to be everywhere during the conference. On the other hand, as the team was big, it was impossible not to see us all over the place. I think for the next Labitconf conference it could be interesting to have a booth and to contrast the experience of the team.
+
+In cases where the event is an expo of more than 4 days with dozens of thousands of attendees, like Web Summit or Talent Land, it is worthwhile to have a physical booth and a big Decred crew.
+
+## Important Interactions
+
+Some of the people and companies we got the chance to talk are:
+
+- Andreas Antonopoulous - when asked about decentralised governance he acknowledged that governance is going to matter more over time as public blockchain grow.
+- Jimmy Song - when asked about decentralised governance he mentioned that governance was unavoidably centralised and that term was in it self an oxymoron.
+- Adam Back - we briefly talk and took a picture with Adam Back.
+- Rodolfo Andragnes organiser of Labitconf and founder of Bitcoin Argentina, we had several conversations about the next Labitconf in Guatemala and the participation of Decred next year.
+- Alejandro de la Torre VP from Poolin, we met and talk about the role of Poolin in Decred mining.
+- Diego Saldivar CEO de RIF labs, founder of RSK.
+- Pablo Coirolo from Aeternity Blockchain.
+- Rocelo Lopes from Stratum.
+- Pablo Gonzales CEO, Eduardo Arenas Listing Manager and Jose Rodriguez VP Payments from Bitso, we talk about having DCR listed next year after they settle their operations in Argentina and Brazil.
+- Tim Byun CEO OKCoin, Jim Nguyen VP Marketing, Oscar Muñoz Compliance Officer, Pablo Almagro Regional Manager Latam, we talk about having a cohosted event between OKCoin and Decred.
+- Rafael Bonventi Client Success Specialist, Lane Fraum Account Manager at Kraken Exchange, talked about having DCR listed and get introduced to the listing team to follow up on the conversation.
```

Comment:
Rafael Bonventi Client Success Specialist, Lane Fraum Account Manager at Kraken Exchange, talked about opportunities between Kraken and DCR and get introduced to the listing team to follow up on the conversation.